<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLineEdit" name="stationOrigin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="stationDestination">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="butZoomIn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>butZoomIn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butZoomOut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>butZoomOut</string>
    </property>
   </widget>
   <widget class="QLabel" name="originNote">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="destinationNote">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>短路径优先</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>少换乘优先</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butFindWay">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>findWay</string>
    </property>
   </widget>
   <widget class="QLabel" name="testOrigin">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起点</string>
    </property>
   </widget>
   <widget class="QLabel" name="testDestination">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>终点</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="LineName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
